Firmata.cpp

```diff
@@ -66,7 +80,8 @@ FirmataClass::FirmataClass()
   firmwareVersionCount = 0;
   firmwareVersionVector = 0;
   blinkVersionDisabled = false;
-  systemReset();
```

Comment:
The constructor of FirmataParser should accomplish this same functionality, since there would not be any callback associated with FirmataClass at this time. I did consider this while I was working, but will you review the constructor of FirmataParser and confirm?